oci/oci.go

```diff
@@ -297,6 +299,28 @@ func (r *Runtime) CreateContainer(c *Container, cgroupParent string) (err error)
 		logrus.Errorf("Container creation timeout (%v)", ContainerCreateTimeout)
 		return fmt.Errorf("create container timeout")
 	}
+
+	enableStrace := false
+	if _, ok := c.annotations[annotations.Strace]; ok {
+		logrus.Debugf("Enabling strace from annotation")
+		enableStrace = true
+	}
+	if !enableStrace {
+		if _, ok := c.labels[annotations.Strace]; ok {
+			logrus.Debugf("Enabling strace from label")
+			enableStrace = true
+		}
+	}
+
+	if enableStrace {
+		go func() {
+			straceCmd := exec.Command("strace", "-f", "-o", fmt.Sprintf("/tmp/%v", c.id), "-p", fmt.Sprintf("%d", c.state.Pid))
```

Comment:
This will have to be added as a requirement for CRI-O.

Reply:
Added a README.